src/components/CardBtn/CardBtn.jsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
```

Comment:
Pls use proptypes validation

Reply:
Of course, I will remove the comments from the code))